scripts/perf/deploy_instance/deploy_instance.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ -z "${CROMWELL_INSTANCE_NAME}" ]
+then
+  echo "[$(date)] Cannot run deploy_instance without CROMWELL_INSTANCE_NAME set"
+  exit 1
+fi
+
+GCS_BUCKET=gs://cromwell-perf-test/
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+DOCKER_ETC_PATH=/usr/share/etc
+
+mkdir -p mnt
+
+DB_PASS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN \
+	broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf | jq '.data.db_pass'`
+
+docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf/service-account-deployer | jq -r '.data.service_account' > mnt/sa.json
```

Comment:
`jq -r --exit-status '.data.service_account'`, or better yet DRY this file out by moving common functions into an include like `helper.inc.sh`.